SDL/Constants.h

```diff
@@ -139,11 +139,11 @@ namespace SDL {
   const unsigned int size_superbins = 45000;
 
   // Temporary fix for endcap buffer allocation.
-  const unsigned int endcap_size = 9105;
+  const unsigned int endcap_size = 9104;
```

Comment:
Just as a note, I think this change was caused by a harmless bug where the old load function would incorrectly load in the top line with the column names of the current endcap file. The new CSV generated endcap file and the current endcap file have the same number of rows (i.e. a sensor didn't just disappear from the endcap).